glider_kinematics_gazebo_plugins/kinematics_control_plugins/src/direct_kinematics.cc

```diff
@@ -0,0 +1,131 @@
+#ifndef _DIRECT_KINEMATICS_HH_
```

Comment:
It is helpful to have a separate header file and include API documentation there.

Here is a good example:
- Header: https://github.com/osrf/vrx/blob/master/usv_gazebo_plugins/include/usv_gazebo_plugins/acoustic_pinger_plugin.hh
- Source: https://github.com/osrf/vrx/blob/master/usv_gazebo_plugins/src/acoustic_pinger_plugin.cc